contents/docs/error-tracking/index.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Error tracking
+---
+
+> ⚠️ Error tracking is considered an alpha product while we continue to ship some crucial features.
+
+Error tracking allows you to track, investigate, and resolve exceptions your customers face.
+
+Errors are captured as `$exception` events which means that you can create insights, filter recordings and trigger surveys based on them exactly the same way you can for any other type of event.
+
+## Installation
+
+### Python
+
+Exception autocapture can be enabled during initialization of the PostHog client to automatically capture any exception thrown by your Python application.
+
+```python
+from posthog import Posthog
+
+posthog = Posthog("API_KEY", enable_exception_autocapture=True, ...)
+```
+
+For exceptions handled by your application that you would still like sent to PostHog it is possible to manually call the capture method.
+
+```python
+posthog.capture_exception(e, 'DISTINCT_ID', properties=additional_properties)
+```
+
```

Comment:
maybe short note about django integration as well?

If you're using Django 4.2+, you can enable the exception autocapture integration which will also automatically capture django errors, like so:

```
from posthoganalytics.exception_capture import Integrations

Posthog("API_KEY", enable_exception_autocapture=True, exception_autocapture_integrations = [Integrations.Django])
```

Reply:
Good shout!